src/util.c

```diff
@@ -1037,6 +1037,21 @@ int ld2string(char *buf, size_t len, long double value, ld2string_mode mode) {
     return 0;
 }
 
+/* Populate the provided seed array by hashing the provided string with SHA1
+ * and copying the first 16 bytes of the digest into the seed buffer. */
```

Comment:
To match the implementation below...

```suggestion
/* Populate the provided seed array by hashing the provided string with SHA256
 * and copying the first outlen bytes of the digest into the seed buffer. */
```